config/docker-compose.yml.tim

```diff
@@ -29,6 +27,4 @@ services:
       - "/bin/sh"
       - "-c"
       - |
-          /opt/sd/launch --api-uri {{api_uri}} --store-uri {{store_uri}} --emitter /opt/sd/emitter {{build_id}} &
-          /opt/sd/logservice --emitter /opt/sd/emitter --api-uri {{api_uri}} --store-uri {{store_uri}} --build {{build_id}} &
-          wait $$(jobs -p)
+          /opt/sd/run.sh "{{token}}" "{{api_uri}}" "{{store_uri}}" "$SD_BUILD_TIMEOUT" "{{build_id}}"
```

Comment:
To template parameter, set values here.
https://github.com/screwdriver-cd/executor-jenkins/blob/master/index.js#L161
This method called from here.
https://github.com/screwdriver-cd/executor-jenkins/blob/master/index.js#L272
However, buildTimeout is calculated here.
https://github.com/screwdriver-cd/executor-jenkins/blob/master/index.js#L276
`SD_BUILD_TIMEOUT` env has to be set. Therefore, to template parameter, some methods have to be fixed to receive buildtimeout value. It is very cumbersome.

Reply:
Is this `SD_BUILD_TIMEOUT` same as https://github.com/screwdriver-cd/executor-jenkins/blob/master/index.js#L276 ? If not there will be inconsistency in Token time out and Jenkins job timeout.

Reply:
Currently, `SD_BUILD_TIMEOUT` env is set in the executor as build parameter.
https://github.com/screwdriver-cd/executor-jenkins/blob/master/index.js#L293
https://github.com/screwdriver-cd/executor-jenkins/blob/master/config/job.xml.tim#L34
And currently, executor-jenkins does not pass buildtimeout as parameter when start launcher process in the build.
https://github.com/screwdriver-cd/executor-jenkins/blob/master/config/docker-compose.yml.tim#L32
Launcher process use `SD_BUILD_TIMEOUT` env because this environment is set as a default value.
https://github.com/screwdriver-cd/launcher/blob/183b9808be3b8686647f2731c8fd259c6b873202/launch.go#L563
So this `SD_BUILD_TIMEOUT` same as https://github.com/screwdriver-cd/executor-jenkins/blob/master/index.js#L276 .